src/test/java/com/arpnetworking/metrics/mad/parsers/GraphitePlaintextToRecordParserTest.java

```diff
@@ -43,6 +47,31 @@
  */
 public final class GraphitePlaintextToRecordParserTest {
 
+    private final Supplier<GraphitePlaintextToRecordParser.Builder> _graphiteParserBuilder =
+            () -> new GraphitePlaintextToRecordParser.Builder()
+                    .setGlobalTags(ImmutableMap.of("host", "localhost"))
+                    .setParseCarbonTags(true);
+
+
+    @Test
+    public void testBuilder() throws InvocationTargetException, IllegalAccessException {
+        BuildableTestHelper.testBuild(
+                _graphiteParserBuilder.get(),
+                GraphitePlaintextToRecordParser.class);
+    }
+
+    @Test
+    public void testReset() throws Exception {
+        ThreadLocalBuildableTestHelper.testReset(_graphiteParserBuilder.get());
+    }
+
+    @Test
+    public void testToString() {
+        final String asString = _graphiteParserBuilder.get().build().toString();
+        Assert.assertNotNull(asString);
+        Assert.assertFalse(asString.isEmpty());
+    }
```

Comment:
I can take a look at adding that to Commons (https://github.com/ArpNetworking/commons/issues/93). I think my only concerns, and they are minor, would be:

1) It violates having a test for something specific.
2) The "~" in verbatim is correct; there are builders that for various reasons behave differently.
3) I think the better approach would be to have a specific test for all builders, instead of a generic test per builder; e.g. the crawler solution we talked about in Commons (https://github.com/ArpNetworking/commons/pull/92#issuecomment-634873997).

Reply:
Fair enough!